Support Engineer Role (#262)
Babylonpartners/ios-playbook: Cookbook/Technical-Documents/SupportEngineerRole.md

# Support Engineer Role

As iOS Engineers, we have Squad related work and we also have iOS Chapter work.

The iOS Team duties are as important as Squad duties, although in some occasions one can have priority over the other.<br>

> When an engineer is in support these duties have high priority over Squad duties, however this can change if there is critical Squad work to be done. In this case the Support Engineer can ask an iOS Chapter Lead to be replaced in his support duties.

Reviewer: ```suggestion
When an engineer is on support, these duties have higher priority over Squad duties, however this can change if there is critical Squad work to be done. In this case, the Support Engineer can ask an iOS Chapter Lead to be replaced in their support duties.
```
```suggestion
When an engineer is in support these duties have high priority over Squad duties, however this can change if there is critical Squad work to be done. In this case the Support Engineer can ask an iOS Chapter Lead to be replaced in his support duties.
```

## Duties

* **Update #ios and #ios-usa-support Slack channels topics** The topics should be updated with the iOS Engineers in support for the current day. This update should be done first thing in the morning.
* **Be available to answer any requests posted in the #ios and #ios-usa-support.** The engineer can either solve the request or delegate to someone who might know how to solve it.
* **Pick up tickets from the iOS Platform backlog.** These are tickets that don't have any Squad as an owner and still need to be fixed. [Kanban board](https://babylonpartners.atlassian.net/secure/RapidBoard.jspa?rapidView=1100&projectKey=IOSP)

> * **Work in personal goals.** Personal goals should be tasks that will help you improve competences in your preferred areas. They are also important tasks that are used, in addition to other metrics, to evaluate your performance. You should only work in these if there is no other task to be picked up from the points defined above and there is no critical work to be done as part of your Squad.

Reviewer: ```suggestion
* **Work on personal goals.** Personal goals should be tasks that will help you improve competences in your preferred areas. They are also important tasks that are used, in addition to other metrics, to evaluate your performance. You should only work on these if there is no other task to be picked up from the points defined above and there is no critical work to be done as part of your Squad.
```

## Schedule

> A support engineer schedule restarts every sprint.<br>

Reviewer: ```suggestion
A support engineer schedule restarts every sprint.
```
